Documentation/ceph-dashboard.md

```diff
@@ -126,3 +126,8 @@ rook-ceph-mgr-dashboard-external-https  NodePort    10.101.209.6     <none>
 In this example, port `31176` will be opened to expose port `8443` from the ceph-mgr pod. Find the ip address
 of the VM. If using minikube, you can run `minikube ip` to find the ip address.
 Now you can enter the URL in your browser such as `https://192.168.99.110:31176` and the dashboard will appear.
+
+To be able to login to the Ceph Dashboard, you will want to set the login credentials:
```

Comment:
@travisn I did see the "credentials" section (lines 45-50) but wasn't able to login with that password, so I went about setting login and password.  It may be useful to mention it, i.e. move the paragraph where the "credentials" section is.

Reply:
Please make it possible to set a custom password that persists.

Reply:
@julienlim Sounds good, so you can move this up to the credentials section?
@vistalba If you call the command in this PR to `ceph dashboard set-login-credentials`, do you not see that it is persistent?

Reply:
@travisn If I set a custom password with 'ceph dashboard set-login-credentials' this is not persistend. If I restart rook-operator or rook-tools (don't remember which one) it is set back to default. I already tried to set it directly in the kubernetes secret but this doesn't work :(